build.gradle

```diff
@@ -0,0 +1,30 @@
+apply plugin: 'groovy'
+
+repositories {
+  mavenCentral()
+}
+
+dependencies {
+  compile 'org.codehaus.groovy:groovy-all:2.4.10'
```

Comment:
does this automatically pull in groovy, or do we have to install it somewhere beforehand?

Reply:
@mst-ableton Unless I'm mistaken, it automatically pulls it in. We don't have Groovy installed on the build nodes and the build worked there.